earth2studio/io/netcdf4.py

```diff
@@ -51,13 +51,33 @@ def __init__(
         # Set persist to false if diskless is false
         self.root = Dataset(
             file_name,
-            "w",
+            "r+",
```

Comment:
Will this fix the issue with running an example twice and it erroring because the file already exists?